### PR TITLE
docs: cli related fixes

### PR DESCRIPTION
**Pull request by alan-agius4**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
back tick is breaking the links

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: aio/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "build-for": "yarn ~~build --configuration",
     "prebuild-local": "yarn setup-local",
     "build-local": "yarn ~~build --configuration=stable",
-    "extract-cli-command-docs": "node tools/transforms/cli-docs-package/extract-cli-commands.js 93bb6efee",
+    "extract-cli-command-docs": "node tools/transforms/cli-docs-package/extract-cli-commands.js 6e7f4d9",
```
Comment (alexeagle) [inline aio/package.json]: ideally we should use the full SHA here since collisions on the 7-digit prefix are possible

**Review by alexeagle — APPROVED** [review]

File: aio/package.json (hunk 1 above)
Comment (alan-agius4) [reply aio/package.json]: Thats actually the tag name https://github.com/angular/cli-builds/releases/tag/6e7f4d9

**Review by jbogarthyde — APPROVED** [review]
"Allowed values given in bold" example doesn't render properly. I suggest removing the example.

**Review by jenniferfell — APPROVED** [review]
LGTM. Two things for the future:

* I'm strongly encouraging everyone to stop submitting and merging "kitchen sink" PRs like this one. There seem to be 4 unrelated activities in this PR: CLI work (a fix to CLI overview, adding CLI to marketing page), changes to alerts that were incorrectly coded (in non-CLI docs), a code correction to the testing doc, and a change to angular.json. This makes the commit history inaccurate, and it makes it harder to review and approve the changes (different reviewers and approvers for different kinds of changes). 

* CLI command names should be in code font. I see that they aren't consistent in the CLI overview, so this change works. We'll submit this doc to our editor for a full copy edit. 

Mostly, however, THANK you for catching and fixing all of this! :-)

**Comment by alan-agius4** [comment]
Couple of follow ups https://github.com/angular/angular/pull/26519 & https://github.com/angular/angular/pull/26517

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**